malwarecage/resources/metakey.py

```diff
@@ -140,6 +140,61 @@ def post(self, type, identifier):
         schema = MetakeyListResponseSchema()
         return schema.dump({"metakeys": metakeys})
 
+    @requires_authorization
+    @requires_capabilities("removing_attributes")
+    def delete(self, type, identifier):
+        """
+        ---
+        summary: Delete object attribute
+        description: |
+            Delete attribute from specified object.
+
+            User must have `removing_attributes` capability.
+        security:
+            - bearerAuth: []
+        tags:
+            - attribute
+        parameters:
+            - in: path
+              name: type
+              schema:
+                type: string
+                enum: [file, config, blob, object]
+              description: Type of object
+            - in: path
+              name: identifier
+              schema:
+                type: string
+              description: Object identifier
+        requestBody:
+            description: Attribute key and value
+            content:
+              application/json:
+                schema: MetakeyItemRequestSchema
+        responses:
+            200:
+                description: When metakey was deleted successfully
+            404:
+                description: |
+                    When object doesn't exist or user doesn't have access to this object.
+                    When attribute key is not defined or user doesn't have privileges to set that one.
+        """
+        schema = MetakeyItemRequestSchema()
+        obj = schema.loads(request.get_data(as_text=True))
+        if obj.errors:
+            return {"errors": obj.errors}, 400
+
+        db_object = access_object(type, identifier)
+        if db_object is None:
+            raise NotFound("Object not found")
+
+        key = obj.data['key']
+        value = obj.data['value']
+        db_object.remove_metakey(key, value)
```

Comment:
This can fail (return `False`) and should return NotFound in that case (see `MetakeyResource.post`)